finda-volunteer/src/main/proto/auth.proto

```
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package auth;
```

Buf (1.66.1)

[error] Files with package "auth" must be within a directory "auth" relative to root but were in directory "finda-volunteer/src/main/proto".

(PACKAGE_DIRECTORY_MATCH)